.env.sample

```
@@ -0,0 +1,13 @@
+DATABASE_URL="mysql://USERNAME:PASSWORD@HOST:TYPEORMPORT/DATABASE"
+
+PORT = YOUR_PORT
+
+TYPEORM_CONNECTION = mysql
+TYPEORM_HOST = YOUR_HOST
+TYPEORM_USERNAME = YOUR_USERNAME
+TYPEORM_PASSWORD = YOUR_PASSWORD
+TYPEORM_DATABASE = YOUR_DATABASE
+TYPEORM_PORT = YOUR_DATABASE_PORT
+TYPEORM_LOGGING = TRUE
+
+secretKey = YOUR SECRET KEY
```

```suggestion
JWT_SECRET_KEY = YOUR SECRET KEY
```
환경 변수는 camelCase가 아닌 다음과 같이 작성될 수 있게 해주세요.

---

네 수정했습니다 !